packages/sdk/src/instance.ts

```diff
@@ -87,6 +87,7 @@ export type DatafileFetchHandler = (datafileUrl: string) => Promise<DatafileCont
 export enum EvaluationReason {
```

Comment:
if you are an SDK developer in other languages, this file change will be of your interest.